src/verb-plugin.js

```diff
@@ -4,65 +4,32 @@
  * Module dependencies.
  */
 
-const wordnet = require('wordnet');
+const extraVerbs = require('./extra-verbs.json');
+const wordnetVerbs = require('./wordnet-verbs.json');
 
 /**
- * Instances & constants.
+ * Constants.
  */
 
-let wordnetInitPromise;
+const allVerbs = { ...extraVerbs, ...wordnetVerbs };
 const normalizeVerbPrefixesRegExp = /^(?:re|un|de)-?(.+?)$/;
-const whitelistedVerbs = [
-  'anonymise',
-  'anonymize',
-  'backfill',
-  'blacklist',
-  'decrement',
-  'dry-run',
-  'increment',
-  'lint',
-  'promisify',
-  'refactor',
-  'setup',
-  'whitelist'
-];
 
 /**
- * Checks if a word is a verb in the simple-present tense.
+ * Checks if a word is a verb according to our dictionaries.
  */
 
-const isVerb = async word => {
-  // If it's whitelisted, simply return true.
-  if (whitelistedVerbs.includes(word)) {
-    return true;
-  }
-
-  let definitions;
-
-  try {
-    definitions = await wordnet.lookup(word);
-  } catch (err) {
-    return whitelistedVerbs.includes(word);
-  }
-
-  const verb = definitions.find(({ meta }) => meta.synsetType === 'verb');
-
-  return !!verb;
-};
+const isVerb = word => !!allVerbs[word] || !!extraVerbs[word];
```

Comment:
```suggestion
const isVerb = word => !!wordnetVerbs[word] || !!extraVerbs[word];
```

or 

```suggestion
const isVerb = word => !!allVerbs[word];
```

Not a mix 😄